File: Work/WK/MS/Management/Management/Management/Management.Web/Modules/SystemSettings/LotteryInfo/LotteryInfoForm.cs
using Serenity.ComponentModel;
using Serenity.Web;

namespace Management.SystemSettings.Forms
{
    [FormScript("SystemSettings.LotteryInfo")]
    [BasedOnRow(typeof(LotteryInfoRow), CheckNames = false)]
    public class LotteryInfoForm
    {
        public string LotteryType { get; set; }
        public string TypeUrl { get; set; }
        public int GameTypeId { get; set; }
        public int Priority { get; set; }
        public string OfficialLotteryUrl { get; set; }
        public string NumberTrendUrl { get; set; }
        public int Status { get; set; }
        public int DefaultSec { get; set; }
        public int AppPriority { get; set; }
        public int HotNew { get; set; }
        public int MaxBonusMoney { get; set; }
        public string Notice { get; set; }
        public int RecommendSort { get; set; }
    }
}

File: Work/WK/MS/Management/Management/Management/Management.Web/Modules/ServerTypes/SystemSettings/LotteryInfoForm.ts
import { StringEditor, IntegerEditor, PrefixedContext } from "@serenity-is/corelib";
import { initFormType } from "@serenity-is/corelib/q";

export interface LotteryInfoForm {
    LotteryType: StringEditor;
    TypeUrl: StringEditor;
    GameTypeId: IntegerEditor;
    Priority: IntegerEditor;
    OfficialLotteryUrl: StringEditor;
    NumberTrendUrl: StringEditor;
    Status: IntegerEditor;
    DefaultSec: IntegerEditor;
    AppPriority: IntegerEditor;
    HotNew: IntegerEditor;
    MaxBonusMoney: IntegerEditor;
    Notice: StringEditor;
    RecommendSort: IntegerEditor;
}

export class LotteryInfoForm extends PrefixedContext {
    static formKey = 'SystemSettings.LotteryInfo';
    private static init: boolean;

    constructor(prefix: string) {
        super(prefix);

        if (!LotteryInfoForm.init)  {
            LotteryInfoForm.init = true;

            var w0 = StringEditor;
            var w1 = IntegerEditor;

            initFormType(LotteryInfoForm, [
                'LotteryType', w0,
                'TypeUrl', w0,
                'GameTypeId', w1,
                'Priority', w1,
                'OfficialLotteryUrl', w0,
                'NumberTrendUrl', w0,
                'Status', w1,
                'DefaultSec', w1,
                'AppPriority', w1,
                'HotNew', w1,
                'MaxBonusMoney', w1,
                'Notice', w0,
                'RecommendSort', w1
            ]);
        }
    }
}
